src/main/java/utility/PrintInfo.java

```diff
@@ -0,0 +1,5 @@
+package utility;
+
+public interface PrintInfo {
```

Comment:
Consider renaming the interface to better reflect its purpose without using a verb, such as 'InfoPrinter' or 'Printable', to follow Java naming conventions.